### PR TITLE
[1LP][WIP] sort out iteritems/keys and most imports for py3 compat preparation

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 015f35915e27fad65069baa55922c7bc5885033a:

`cfme/tests/configure/test_db_backup_schedule.py`:
- :red_circle: Line 15:1: **F401** *'six' imported but unused*

`cfme/tests/containers/test_logging.py`:
- :bangbang: Line 36:101: **E501** *line too long (103 > 100 characters)*

`cfme/tests/containers/test_manageiq_ansible_tags.py`:
- :bangbang: Line 67:101: **E501** *line too long (105 > 100 characters)*

`cfme/tests/containers/test_table_views.py`:
- :bangbang: Line 44:101: **E501** *line too long (101 > 100 characters)*
- :bangbang: Line 47:101: **E501** *line too long (103 > 100 characters)*

`cfme/utils/appliance/__init__.py`:
- :bangbang: Line 1871:101: **E501** *line too long (102 > 100 characters)*

`cfme/utils/perf_message_stats.py`:
- :bangbang: Line 356:101: **E501** *line too long (106 > 100 characters)*

`scripts/release.py`:
- :bangbang: Line 101:101: **E501** *line too long (101 > 100 characters)*

`scripts/sync_template_tracker.py`:
- :bangbang: Line 127:101: **E501** *line too long (103 > 100 characters)*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

File: sprout/logserver.py [hunk 1]
```diff
@@ -1,14 +1,16 @@
 #!/usr/bin/env python2
```
[the remark this answers is not in the document]
Comment (RonnyPfannschmidt) [reply sprout/logserver.py]: this pr isn't supposed to sort out that bit, the main focus are imports and iteritems/keys/values in order to massively ease future work

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by RonnyPfannschmidt** [comment]
closing due to repeat rebase issues

**Pull request closed without merging**